test/e2e/tests/01-apply-recommendations/02-wait-for-controller.yaml

```
@@ -0,0 +1,5 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - command: sleep 90
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)